cypress/integration/v2/blocks/card.spec.js

```diff
@@ -58,3 +63,403 @@ function switchDesign() {
 	] ) )
 }
 
+function styleTab( viewport, desktopOnly ) {
+	cy.setupWP()
+	cy.newPage()
+	cy.addBlock( 'ugb/card' )
+	cy.openInspector( 'ugb/card', 'Style' )
+
+	cy.collapse( 'General' )
+	desktopOnly( () => {
+		cy.adjust( 'Columns', 3 )
+			.assertClassName( '.ugb-card', 'ugb-card--columns-3' )
+	} )
+	assertAligns( 'Align', '.ugb-inner-block', { viewport } )
+
+	cy.collapse( 'Container' )
+
+	desktopOnly( () => {
+		cy.adjust( 'Background', {
+			[ `Color Type` ]: 'single',
+			[ `Background Color` ]: '#000000',
+			[ `Background Color Opacity` ]: 0.7,
+		} ).assertComputedStyle( {
+			'.ugb-card__item': {
+				[ `background-color` ]: 'rgba(0, 0, 0, 0.7)',
+			},
+		} )
+	} )
+
+	cy.setBlockAttribute( {
+		[ `column${ viewport === 'Desktop' ? '' : viewport }BackgroundMediaUrl` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+		[ `image1Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+		[ `image2Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+		[ `image3Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
```

Comment:
`image3Url` has no effect in tablet and mobile. We can do:
1) remove the `image3Url` here, or
2) Move the `cy.adjust( 'Columns', 3 )` outside of `desktopOnly`